Web/Element/HeroImage.hs

```diff
@@ -0,0 +1,55 @@
+module Web.Element.HeroImage where
+
+import Web.View.Prelude
+
+import Application.Helper.Icons
+import Web.Element.ElementWrap
+import Web.Element.Types
+import Web.Element.Button
+
+render :: RenderHeroImage -> Html
+render record@RenderHeroImage {title, subtitle, button, imageUrl} =
+     titleWrapped
+        ++ subTitleWrapped
+        ++ buttonHtml
+        |> wrapVerticalSpacingBig AlignNone
+        |> renderImageAndContent (pathTo $ RenderImageStyleAction 1536 466 imageUrl signed)
```

Comment:
What are the numbers `1536 466`? Worth a comment

Reply:
It's image size, added comment